lib/screens/profile.dart

```diff
@@ -0,0 +1,272 @@
+                    color: ThemeColoursSeva().pallete1,
+                    textColor: Colors.white,
+                  );
+                } else
+                  return Container();
+              },
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          "My Account",
+          style: TextStyle(
+              color: ThemeColoursSeva().pallete1, fontWeight: FontWeight.w500),
+        ),
+        backgroundColor: ThemeColoursSeva().pallete3,
+        centerTitle: true,
+        elevation: 1.0,
+        leading: IconButton(
+          icon: Icon(
+            Icons.arrow_back,
+            color: Colors.black54,
+          ),
+          onPressed: () => Navigator.of(context).pop(),
+        ),
+        actions: [CartIcon()],
+      ),
+      // This is for the scrolling effect
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            Container(
+              decoration: BoxDecoration(color: ThemeColoursSeva().pallete3),
+              height: 200,
+              child: Padding(
+                padding: const EdgeInsets.only(top: 8.0),
+                child: Center(
+                    child: Column(
+                  children: [
+                    Icon(
+                      Icons.account_circle,
+                      color: Colors.black54,
+                      size: 70,
+                    ),
+                    SizedBox(
+                      height: 10,
+                    ),
+                    Text(
+                      widget.username,
+                      style: TextStyle(
+                          fontSize: 2.4 * SizeConfig.textMultiplier,
+                          color: ThemeColoursSeva().pallete1),
+                    ),
+                    SizedBox(
+                      height: 20,
+                    ),
+                    Text(
+                      widget.mobile,
+                      style: TextStyle(
+                          fontSize: 2.1 * SizeConfig.textMultiplier,
+                          color: ThemeColoursSeva().pallete1),
+                    ),
+                    SizedBox(
+                      height: 10,
+                    ),
+                    Text(
+                      widget.email,
+                      style: TextStyle(
+                          fontSize: 2.1 * SizeConfig.textMultiplier,
+                          color: ThemeColoursSeva().pallete1),
+                    )
+                  ],
+                )),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Card(
+                child: Column(
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    const ListTile(
+                      title: Text('My Orders'),
```

Comment:
why orders is here ? They must click 3 times to reach to **Orders** Page if it is here. Should this not be in `sidenav` ?

Reply:
Orders will be there on both the screens otherwise the my account section will look empty